Assets/HoloToolkit/Input/Scripts/Gaze/GazePointer.cs

```diff
@@ -8,35 +8,11 @@
 namespace HoloToolkit.Unity.InputModule
 {
     /// <summary>
-    /// The gaze manager manages everything related to a gaze ray that can interact with other objects.
+    /// Gaze handles everything related to a gaze ray that can interact with other objects.
     /// </summary>
-    public class GazeManager : Singleton<GazeManager>, IPointingSource
+    public class GazePointer : Singleton<GazePointer>, IPointingSource
```

Comment:
I think the rename is not needed. The usual pattern for singletons is the manager suffix.